projects/plugins/jetpack/extensions/plugins/payments/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { ExternalLink } from '@wordpress/components';
+import { PluginPostPublishPanel } from '@wordpress/edit-post';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import analytics from '../../../_inc/client/lib/analytics';
+import useAutosaveAndRedirect from '../../shared/use-autosave-and-redirect/index';
+
+const PaymentsPostPublish = () => {
+	const paymentInfoUrl = 'https://wordpress.com/payments-donations/';
+	const { autosaveAndRedirect } = useAutosaveAndRedirect( paymentInfoUrl );
+	const trackClick = event => {
+		event.preventDefault();
+		analytics.tracks.recordEvent( 'jetpack_editor_payments_post_publish_click' );
+		autosaveAndRedirect( event );
+	};
+
+	return (
+		<PluginPostPublishPanel
+			className="jetpack-payments-post-publish-panel"
+			title={ __( 'Start accepting payments', 'jetpack' ) }
+			initialOpen
+		>
```

Comment:
Just a random idea, good for a follow-up.
We can add customize the panel icon with the `icon` prop.

<img width="271" alt="Screenshot 2022-03-01 at 12 36 00" src="https://user-images.githubusercontent.com/2070010/156170077-5a906fa4-e960-4f71-a019-b99a6e4d7fa3.png">

Reply:
I like the idea, thank you. Done in 8d432060ad3b1f043d66a91c0d1968c7ea171414
<img width="281" alt="Screenshot 2022-03-03 at 15 52 44" src="https://user-images.githubusercontent.com/426388/156589438-a4b23877-a696-4c93-a67f-4ed9a8256f00.png">